spec/lib/myfinance/resources/entity_spec.rb

```diff
@@ -60,4 +60,107 @@
+        expect(subject.id).to be_nil
+        expect(subject.name).to be_nil
+        expect(subject.account_id).to eq(3613)
+        expect(subject.charging_uuid).to be_nil
+        expect(subject.created_at).to be_nil
+        expect(subject.default_in_menu).to be_falsy
+        expect(subject.deleted_at).to be_nil
+        expect(subject.federation_subscription_number).to be_nil
+        expect(subject.imported_from_sync_at).to be_nil
+        expect(subject.updated_at).to be_nil
+      end
+    end
+
+    context "when not found" do
+      let(:client) { Myfinance.client('') }
+      subject { client.entities.get_new }
+
+      it "raises NotFound" do
+        expect{ subject }.to raise_error(Myfinance::RequestError)
+      end
+    end
+  end
+
+  describe "#create", vcr: true do
+    let(:params) do
+      {
+        entity: {
+          name: "Finanças"
+        }
+      }
+    end
+
+    context "when success" do
+      subject { client.entities.create(params) }
+
+      it "returns a created entity" do
+        expect(subject.class).to eq(Myfinance::Entities::Entity)
+        expect(subject.id).to eq(3828)
+        expect(subject.name).to eq("Finanças")
+        expect(subject.account_id).to eq(3613)
+        expect(subject.charging_uuid).to be_nil
+        expect(subject.created_at).to eq(DateTime.parse("2015-10-15T16:11:08-03:00"))
+        expect(subject.default_in_menu).to be_falsy
+        expect(subject.deleted_at).to be_nil
+        expect(subject.federation_subscription_number).to be_nil
+        expect(subject.imported_from_sync_at).to be_nil
+        expect(subject.updated_at).to eq(DateTime.parse("2015-10-15T16:11:08-03:00"))
+      end
+    end
+
+    context "when not found" do
+      let(:client) { Myfinance.client('') }
+      subject { client.entities.create(params) }
+
+      it "raises NotFound" do
+        expect{ subject }.to raise_error(Myfinance::RequestError)
+      end
+    end
+  end
+
+  describe "#update", vcr: true do
+    let(:params) do
+      {
+        entity: {
+          name: "Finanças 2"
+        }
+      }
+    end
+
+    context "when success" do
+      subject { client.entities.update(3828, params) }
+
+      it "returns a updated entity" do
+        expect(subject.class).to eq(Myfinance::Entities::Entity)
+        expect(subject.id).to eq(3828)
+        expect(subject.name).to eq("Finanças 2")
+        expect(subject.account_id).to eq(3613)
+        expect(subject.charging_uuid).to be_nil
+        expect(subject.created_at).to eq(DateTime.parse("2015-10-15T16:11:08-03:00"))
+        expect(subject.default_in_menu).to be_falsy
+        expect(subject.deleted_at).to be_nil
+        expect(subject.federation_subscription_number).to be_nil
+        expect(subject.imported_from_sync_at).to be_nil
+        expect(subject.updated_at).to eq(DateTime.parse("2015-10-15T16:11:08-03:00"))
+      end
+    end
+
+    context "when not found" do
+      let(:client) { Myfinance.client('') }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.